src/components/MultiGestureCanvas/index.tsx

```diff
@@ -130,7 +130,7 @@ function MultiGestureCanvas({
 
         if (animated) {
             offsetX.value = withSpring(0, SPRING_CONFIG);
-            offsetY.value = withSpring(0, SPRING_CONFIG);
+            offsetY.value = 0;
```

Comment:
The accepted proposal contained a different fix 
`offsetY.value = withSpring(0, {...SPRING_CONFIG, duration: 0});`
Is there a reason you've decided to use `offsetY.value = 0`?
(Just want to understand this better, this does work fine)

Reply:
hi @eVoloshchak, my apologies for modifying the approved proposal. 
 
The withSpring(0, {...SPRING_CONFIG, duration: 0}) function is causing a typing error. Given that withSpring is designed for generating animated effects, specifying duration: 0 essentially eliminates the animation entirely.

I've made changes to directly set offsetY.value = 0 instead, which promptly reduces the value to 0 with the same outcome (no animation).